src/bos/operators/base.py

```diff
@@ -192,6 +192,8 @@ def _update_database(self, components: List[dict], additional_fields: dict=None)
             if 'desired_state' in patch and 'session' not in patch:
                 raise MissingSessionData
             data.append(patch)
+        LOGGER.info('Found %d components that require updates', len(data))
+        LOGGER.debug(f'Updated components: {data}')
```

Comment:
This could be 10,000 components. I realize it's DEBUG, but this could swamp the logs.

Reply:
Yeah -- we already have other DEBUG statements which do this is the only reason I figured it was considered okay.

Reply:
For example:
* https://github.com/Cray-HPE/bos/blob/3a4ab7cff559a0247859ca78f5d382463e4bc60b/src/bos/operators/actual_state_cleanup.py#L67
* https://github.com/Cray-HPE/bos/blob/3a4ab7cff559a0247859ca78f5d382463e4bc60b/src/bos/operators/filters/base.py#L63
* https://github.com/Cray-HPE/bos/blob/3a4ab7cff559a0247859ca78f5d382463e4bc60b/src/bos/operators/status.py#L96